content/spire/try/oidc_federation_aws.md

```diff
@@ -0,0 +1,399 @@
+
+In a typical deployment, you would configure one machine to run the SPIRE Server and OIDC Discovery Provider and a second machine to run the SPIRE Agent. The machines can be a physical machine, virtual machine, instance, or container. The examples in these instructions do not describe how to use Kubernetes or Docker commands to set up OIDC federation, but they are supported.
+
+Review the [requirements](#required-accounts-and-permissions) described above.
+
+## Install, Configure, and Run the SPIRE Server
+
+The steps below provide an overview of installing and configuring the SPIRE Server. For more detailed information, see [Install SPIRE Server](/spire/docs/install-server/).
+
+1. Set up an instance or machine on which to run the SPIRE Server and OIDC Discovery Provider.
+
+2. [Download](/downloads/) and [install](/spire/docs/install-server/) SPIRE Server.
+
+3. Modify the following two settings in your `server.conf` SPIRE Server configuration file.
+
+| Setting | Example | Description |
+| --- | --- | --- |
+| `ca_key_type` |  `rsa-2048` | The key type used for the server certificate authority.<br/> Valid key types depend on the service to which you are configuring federation. For example, to authenticate to AWS services, the only valid key type is `rsa-2048`. AWS does not support elliptical curve keys.|
+| `jwt_issuer` | `https://spire-oidc.example.org` | The URL to the OIDC discovery-supporting identity provider. This value is put in the "iss" (issuer) claim in JWT-SVIDs created by SPIRE Server.  This should be the same domain that you configure for the `domain` setting in the  OIDC Discovery Provider `oidc-discovery-provider.conf` file. |
+
+See [Configuring SPIRE](/spire/docs/configuring) and [Server configuration file](https://github.com/spiffe/spire/blob/master/doc/spire_server.md#server-configuration-file) for more information about SPIRE Server configuration settings.
+
+4. When initially configuring OIDC federation, you may want to use the Join Token method of [node attestation](/spire/concepts#node-attestation). If so, add the following lines to your `server.conf` file:
+
+```hcl
+NodeAttestor "join_token" {
+    plugin_data {
+    }
+}
+```
+
+5. Make any other necessary changes to your SPIRE Server configuration file such as configuring a trust domain, bind port, and data store as described in [Configuring SPIRE](/spire/docs/configuring/). For more information on `server.conf` options, see the
+[SPIRE Server Configuration Reference](https://github.com/spiffe/spire/blob/master/doc/spire_server.md#server-configuration-file) on GitHub.
+
+6. Start the SPIRE server. For example, on Linux, run
+```
+sudo spire-server run  -config /opt/spire/conf/server/server.conf
+```
+
+## Install, configure, and run the OIDC Discovery Provider
+
+The SPIRE OIDC Discovery Provider serves OIDC discovery documents via a REST API. See the separate [OIDC Discovery Provider](/spire/docs/oidc_discovery/) installation documentation.
```

Comment:
We shouldn't do this - since the `/spire/docs/oidc_discovery/` is general reference documentation but the instructions we give here should be specific to this tutorial (including configuration). We should describe, in this document, how to solve for the problem step by step.